rootfs/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.5
+RUN apt-get upgrade && apt-get install -y && curl -sSL https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.py | python - && pip install flake8 
```

Comment:
This is a good practice to combine multiple commands into a single `RUN` directive wherever you can.  :+1: 

For improved readability you might want to consider inserting the line continuation character `\` and a line break between commands (i.e. after each `&&`).